### PR TITLE
fix(dedicated): add description on several section

### PR DESCRIPTION
**Pull request by ghyenne**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-8912
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
I added description on several section of manager

### :flags: Translations

- [x] TRANS-43777
- [x] TRANS-44076

### CODE REVIEW

**Review by JeremyDec — APPROVED** [review]

**Review by anooparveti — DISMISSED** [review]

File: packages/manager/apps/dedicated/client/app/account/user/user.html [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <div class="user-account">
-    <oui-header data-heading="{{ :: 'user_account_title_infos' | translate }}">
+    <oui-header
+        data-heading="{{ :: 'user_account_title_infos' | translate }}"
+        data-description="{{ :: $ctrl.coreConfig.getRegion() !== 'US' ? 'user_account_description' : '' | translate }}"
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/account/user/user.html:4]: Could this condition been defined as a feature flipping rule? It is used 4 times

Comment (ghyenne) [reply packages/manager/apps/dedicated/client/app/account/user/user.html:4]: @JeremyDec  In the future there will probably also be a description for US but different from EU and CA (as not the same functions are available).

Comment (radireddy) [reply packages/manager/apps/dedicated/client/app/account/user/user.html:4]: Question: What is the description shown for US region?

File: packages/manager/modules/billing/src/autoRenew/autorenew.html [hunk 2]
```diff
@@ -1,4 +1,7 @@
-<oui-header data-heading="{{:: 'billing_title' | translate}}">
+<oui-header
+    data-heading="{{:: 'billing_title' | translate}}"
+    data-description="{{:: $ctrl.coreConfig.getRegion() !== 'US' ? 'billing_description' : '' | translate}}"
```
Comment (radireddy) [inline packages/manager/modules/billing/src/autoRenew/autorenew.html:3]: Question: Empty heading for US? is it as per the requirements?

File: packages/manager/modules/billing/src/main/billing-main.html [hunk 3]
```diff
@@ -1,11 +1,14 @@
-<oui-header heading="{{:: 'billing_main_title' | translate }}">
+<oui-header
+    data-heading="{{:: 'billing_main_title' | translate }}"
+    data-description="{{:: $ctrl.coreConfig.getRegion() !== 'US' ? 'billing_main_description' : '' | translate }}"
```
Comment (radireddy) [inline packages/manager/modules/billing/src/main/billing-main.html:3]: Question: Empty heading for US? is it as per the requirements?

File: packages/manager/apps/dedicated/client/app/account/user/user.html (hunk 1 above)
Comment (ghyenne) [reply packages/manager/apps/dedicated/client/app/account/user/user.html:4]: It is planned in the future to have US content, but it is not yet adapted and ready.

File: packages/manager/modules/billing/src/autoRenew/autorenew.html (hunk 2 above)
Comment (ghyenne) [reply packages/manager/modules/billing/src/autoRenew/autorenew.html:3]: It is planned in the future to have US content, but it is not yet adapted and ready.

File: packages/manager/modules/billing/src/main/billing-main.html (hunk 3 above)
Comment (ghyenne) [reply packages/manager/modules/billing/src/main/billing-main.html:3]: It is planned in the future to have US content, but it is not yet adapted and ready.

**Review by anooparveti — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/account/user/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -600,6 +600,8 @@
   "user_account_security_double_auth_type_backup_code_delete_error": "Oups ! Une erreur est survenue lors de la suppression des codes de secours :",
   "user_account_security_ip_restrictions_alt": "Restriction d’accès par IP",
   "user_account_title_infos": "Mon profil",
+  "user_account_description": "Administrez votre compte client. Dans cette rubrique, vous gérez la sécurité de votre compte, votre niveau de support et vos utilisateurs. Vous pouvez également consulter les e-mails reçus liés à vos services.",
+  "user_account_description_US": "",
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/account/user/translations/Messages_fr_FR.json:604]: Empty, do we need it ?

File: packages/manager/modules/billing/src/autoRenew/autorenew.controller.js [hunk 5]
```diff
@@ -87,6 +87,14 @@ export default class AutorenewCtrl {
     }
   }
 
+  descriptionOfHeading() {
+    const getRegion = this.coreConfig.getRegion();
```
Comment (mohammed-zahaf) [inline packages/manager/modules/billing/src/autoRenew/autorenew.controller.js:91]: better like this:
```suggestion
    const currentRegion = this.coreConfig.getRegion();
```

File: packages/manager/modules/billing/src/autoRenew/autorenew.controller.js [hunk 6]
```diff
@@ -87,6 +87,14 @@ export default class AutorenewCtrl {
...
+    return this.$translate.instant(`billing_description_${getRegion}`) !==
```
Comment (mohammed-zahaf) [inline packages/manager/modules/billing/src/autoRenew/autorenew.controller.js:92]:
```suggestion
    return this.$translate.instant(`billing_description_${currentRegion}`) !==
```

File: packages/manager/modules/billing/src/autoRenew/autorenew.controller.js [hunk 7]
```diff
@@ -87,6 +87,14 @@ export default class AutorenewCtrl {
...
+      `billing_description_${getRegion}`
+      ? this.$translate.instant(`billing_description_${getRegion}`)
```
Comment (mohammed-zahaf) [inline packages/manager/modules/billing/src/autoRenew/autorenew.controller.js:94]:
```suggestion
      ? this.$translate.instant(`billing_description_${currentRegion}`)
```

File: packages/manager/modules/billing/src/main/billing-main.controller.js [hunk 8]
```diff
@@ -1,7 +1,16 @@
...
   /* @ngInject */
-  constructor(coreConfig, isPayAsYouGoAvailable) {
+  constructor(coreConfig, isPayAsYouGoAvailable, $translate) {
     this.coreConfig = coreConfig;
     this.isPayAsYouGoAvailable = isPayAsYouGoAvailable;
+    this.$translate = $translate;
+  }
+
+  descriptionOfHeading() {
+    const getRegion = this.coreConfig.getRegion();
```
Comment (mohammed-zahaf) [inline packages/manager/modules/billing/src/main/billing-main.controller.js:10]: Same as here:
https://github.com/ovh/manager/pull/6688/files#r865634536

File: packages/manager/apps/dedicated/client/app/account/user/user.controller.js [hunk 9]
```diff
@@ -44,4 +55,12 @@ export default class UserAccountCtrl {
       }
     }.bind(this);
   }
+
+  descriptionOfHeading() {
+    const getRegion = this.coreConfig.getRegion();
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/account/user/user.controller.js:60]: Same as here:
https://github.com/ovh/manager/pull/6688/files#r865634536

Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/account/user/user.controller.js:60]: Regarding repetitive ask for `getRegion`, consider to create a `currentRegion` variable into constructor.

File: packages/manager/modules/billing/src/orders/billing-orders.controller.js [hunk 10]
```diff
@@ -41,6 +43,14 @@ export default class BillingOrdersCtrl {
     this.allowOrderTracking = billingFeatureAvailability.allowOrderTracking();
   }
 
+  descriptionOfHeading() {
+    const getRegion = this.coreConfig.getRegion();
```
Comment (mohammed-zahaf) [inline packages/manager/modules/billing/src/orders/billing-orders.controller.js:47]: Same as here:
https://github.com/ovh/manager/pull/6688/files#r865634536

File: packages/manager/modules/support/src/support.controller.js [hunk 11]
```diff
@@ -19,4 +21,13 @@ export default class {
   stopLoading() {
     this.isLoading = false;
   }
+
+  descriptionOfHeading() {
+    const getRegion = this.coreConfig.getRegion();
```
Comment (mohammed-zahaf) [inline packages/manager/modules/support/src/support.controller.js:26]: To here:
https://github.com/ovh/manager/pull/6688/files#r865634536

File: packages/manager/modules/billing/src/autoRenew/autorenew.controller.js (hunk 5 above)
Comment (ghyenne) [reply packages/manager/modules/billing/src/autoRenew/autorenew.controller.js]: It's applied

File: packages/manager/modules/billing/src/autoRenew/autorenew.controller.js (hunk 6 above)
Comment (ghyenne) [reply packages/manager/modules/billing/src/autoRenew/autorenew.controller.js]: It's applied

File: packages/manager/modules/billing/src/autoRenew/autorenew.controller.js (hunk 7 above)
Comment (ghyenne) [reply packages/manager/modules/billing/src/autoRenew/autorenew.controller.js]: It's applied

File: packages/manager/modules/billing/src/main/billing-main.controller.js (hunk 8 above)
Comment (ghyenne) [reply packages/manager/modules/billing/src/main/billing-main.controller.js]: It's applied

File: packages/manager/apps/dedicated/client/app/account/user/user.controller.js (hunk 9 above)
Comment (ghyenne) [reply packages/manager/apps/dedicated/client/app/account/user/user.controller.js]: It's applied

**Review by radireddy — APPROVED** [review]

File: packages/manager/modules/billing/src/main/billing-main.controller.js [hunk 12]
```diff
@@ -1,7 +1,18 @@
 export default class BillingMainCtrl {
   /* @ngInject */
-  constructor(coreConfig, isPayAsYouGoAvailable) {
+  constructor(coreConfig, isPayAsYouGoAvailable, $translate) {
```
Comment (mohammed-zahaf) [inline packages/manager/modules/billing/src/main/billing-main.controller.js:3]: We have as habit to put angular deps first.

Comment (mohammed-zahaf) [inline packages/manager/modules/billing/src/main/billing-main.controller.js:3]: We have as habit to put angular deps first.
```suggestion
  constructor(translate, coreConfig, isPayAsYouGoAvailable) {
```

**Review by mohammed-zahaf — DISMISSED** [review]

**Review by vks-ovh — DISMISSED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by radireddy — APPROVED** [review]

**Comment by JeremyDec** [comment]
Hello @ghyenne missing translation for packages/manager/apps/dedicated/client/app/account/user/translations/Messages_fr_FR.json ?

**Review by antleblanc — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**